lib/utils.js

```diff
@@ -0,0 +1,15 @@
+export function asyncIterable () {
+    let i = 0;
+    return {
+        next: () => {
+            return new Promise((resolve) => {
+                setTimeout(() => {
+                    resolve({
+                        value: this[i],
+                        done: i++ === this.length,
+                    });
+                }, 0);
+            });
+        },
+    };
```

Comment:
why 4 spaces?

Reply:
My bad! The development environment was not set up.